### ISSUE TITLE
Scheduled view as additional sidebar

### ISSUE THREAD

**Issue by fms-all** [alainm23/planify#1332]
**Is your feature request related to a problem? Please describe.**
I want to plan my tasks for the next week. But I always have to switch back and forth between the “Scheduled” tab (to check for free slots) and the project (to decide which task I want to schedule).

**Describe the solution you'd like**
Possibility to show the scheduled view as an additional sidebar. This way you would always have all the information you need at a glance.

**Describe alternatives you've considered**
Na

**Additional context**
See example screenshot
![PlanifyScheduleSidebar](https://github.com/alainm23/planify/assets/147617751/8c9d1c31-2388-4aec-9a87-0e022d635de1)

**Comment by Androz2091** [comment]
I agree, navigation could be improved (I don't know if it's the best way though). I like keeping the app always opened but there is no view that works great for everything (for instance I like to always see the schedule and it's great to be able to filter with some projects, but why are today's tasks not showing up in the schedule?) 
As mentioned by the author of the issue, I always have to switch back and forth between views

(take it as an improvement though, I really enjoy the app!)

**Comment by alainm23** [comment]
@fms-all I will consider this as an improvement, maybe we could add a next week preview widget, task indicators in the date picker or have the upcoming view as a separate sale.

**Comment by alainm23** [comment]
It looks like an interesting solution but it is not quite complete, there are settings (online accounts, preferences and inbox project id) that are not saved in database but using Gschema.
